### ISSUE TITLE
testing 3784: alternative E-Mail not accessible

### ISSUE THREAD

**Issue by CarolingerSeilchenspringer** [liqd/a4-meinberlin#3494]
**URL:** https://meinberlin-dev.liqd.net/budgeting/2021-00529/update/
**user:** logged in
**expected behaviour:** I can enter an alternative e-mail-address without mouse
**behaviour:** the focus jumps from user account mail to telephone number (should work with tab, right?)  
**important screensize:**
**device & browser:** big sur, chrome
**Comment/Question:** 

Screenshot?

**Comment by Rineee** [comment]
@CarolingerSeilchenspringer You need to select the radio button ' Bitte kontaktieren Sie mich über eine andere E-Mail-Adresse:' and then the textfield can be accessed via tabbing.

**Comment by CarolingerSeilchenspringer** [comment]
@Rineee If my focus is on the first radio button and I press tab, it should jump to the second radio button "andere E-Mail-Adresse", and if I chose this I can jump to the address field, right? It doesn't, it jumps from the first radio button right to the telephone number.

**Comment by Rineee** [comment]
@CarolingerSeilchenspringer No, the selection of radio buttons doesnt work with tabbing but with arrows. So when your focus is on the first radio button, you press 'arrow down' to select the second one and then the text field to enter your new e-mail address can be accessed by tabbing. Does that work?

**Comment by CarolingerSeilchenspringer** [comment]
@Rineee then everything is in perfect order and I learned more about accessibility! Thanks!
